test/spec/utils.ex

```diff
@@ -135,16 +141,35 @@ defmodule SpecTestUtils do
   def sanitize_ssz(bitlist, {:bitlist, _size} = _schema), do: elem(BitList.new(bitlist), 0)
   def sanitize_ssz(bitvector, {:bitvector, size} = _schema), do: BitVector.new(bitvector, size)
 
+  def sanitize_ssz(
```

Comment:
careful with this "sanitation", it's probably producing unintended bugs

We need to ensure that `SszEx.decode(SszEx.encode(container)) == container` for every container.